File: ass59/Program2.java
/*
2. Write Java program which accept String from user and display below pattern. 
Input :  Hello 
Output : H e l l o   
         H e l l    
         H e l   
         H e   
         H
*/
import java.lang.*;
import java.util.*;
class DemoString
{
	int WordFrequency(String str)
	{
		char arr[] = str.toCharArray();
		int size = arr.length;
        int i = 0, j= 0;
        
        for(i=size-1; i>=0; i--)
        {
            for(j=0; j<=i; j++)
            {
                System.out.print(""+arr[j]);
            }
            System.out.print("\n");
        }
        return 0;
    }
    
}
class Program2
{
	public static void main(String arg[])
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter string");

        String str = sobj.nextLine();
        sobj.close();
		DemoString dobj = new DemoString();

		dobj.WordFrequency(str);
	}
}

File: ass45/program2.c
/*
2. Write a recursive program which display below pattern.  
Input :   5 
Output :   1 2 3 4 5 
*/
#include<stdio.h>
void Display(int x)
{
    static int i=1;
    if(i<=x)
    {
        printf("%d",i);
        i++;
        Display(x);
    }
}
int main()
{
    int iNo = 0;
    printf("Enter the Element\n");
    scanf("%d",&iNo);

    Display(iNo);
   
    return 0;
}